src/main/java/model/Car.java

```diff
@@ -0,0 +1,66 @@
+package model;
+
+import model.exception.CarCreateException;
+import model.result.MoveResult;
+
+public class Car implements Comparable<Car> {
+    private static final int CAR_NAME_MAX_LENGTH = 5;
+    public static final int DEFAULT_POSITION = 0;
+
+    private String name;
+    private int position;
+
+    public Car(String name) {
+        this(name, DEFAULT_POSITION);
+    }
+
+    public Car(String name, int position) {
+        validateName(name);
+        this.name = name;
+        this.position = position;
+    }
+
+    private void validateName(String name) {
+        validateNull(name);
+        validateEmpty(name);
+        validateLength(name);
+    }
+
+    private void validateNull(String name) {
+        if (name == null) {
+            throw new CarCreateException("차 이름에 NULL 을 입력할 수 없습니다.");
+        }
+    }
+
+    private void validateEmpty(String name) {
+        if ("".equals(name)) {
+            throw new CarCreateException("차 이름에 비어있는 문자열을 입력할 수 없습니다.");
+        }
+    }
+
+    private void validateLength(String name) {
+        if (name.length() > CAR_NAME_MAX_LENGTH) {
+            throw new CarCreateException("차 이름은 5자 이하이어야합니다.");
```

Comment:
```suggestion
            throw new CarCreateException("차 이름은 " + CAR_NAME_MAX_LENGTH + "자 이하이어야합니다.");
```
저도 똑같이 실수한 부분인데, 메시지를 상수와 함께 맞춰준다면 나중에 변경할 때 유용할거 같아요.

Reply:
다른 메시지에도 적용할 수 있는 곳이 많네요